packages/env/lib/create-docker-compose-config.js

```diff
@@ -1,24 +1,25 @@
 module.exports = function createDockerComposeConfig(
 	cwdTestsPath,
 	context,
-	dependencies,
+	dependencies
 ) {
 	const { path: cwd, pathBasename: cwdName } = context;
 
-	const dependencyMappings = [ ...dependencies, context ].map(
-		( { path, pathBasename, type } ) => `      - ${ path }/:/var/www/html/wp-content/${ type }s/${ pathBasename }/\n`
-	).join( '' );
+	const dependencyMappings = [ ...dependencies, context ]
+		.map(
+			( { path, pathBasename, type } ) =>
+				`      - ${ path }/:/var/www/html/wp-content/${ type }s/${ pathBasename }/\n`
+		)
+		.join( '' );
 	const commonVolumes = `
 ${ dependencyMappings }
       - ${ cwd }${ cwdTestsPath }/e2e-tests/mu-plugins/:/var/www/html/wp-content/mu-plugins/
       - ${ cwd }${ cwdTestsPath }/e2e-tests/plugins/:/var/www/html/wp-content/plugins/${ cwdName }-test-plugins/`;
 	const volumes = `
       - ${ cwd }/../${ cwdName }-wordpress/:/var/www/html/${ commonVolumes }`;
 	const testsVolumes = `
-      - tests-wordpress:/var/www/html/${ commonVolumes }`;
+      - ${ cwd }/../${ cwdName }-tests-wordpress/:/var/www/html/${ commonVolumes }`;
```

Comment:
I know it exists already for `gutenberg-wordpress`, but this directory creation one level up from the project root was very unexpected for me the first time I encountered it. Was there a reason to not include it as a directory within the project, like already exists with `wordpress`? For one thing, it makes future clean-up easier, since it's kept self-contained.

Reply:
It would make the plugin volume recursive and sometimes Docker tripped up on that for some reason.